include/tvm/tir/builtin.h

```diff
@@ -909,6 +909,11 @@ TVM_DLL const Op& anylist_setitem_call_packed();
  */
 TVM_DLL const Op& anylist_setitem_call_cpacked();
 
+/*!
+ * \brief Get the target's vscale value
```

Comment:
Is it worth adding a link to the LLVM lang ref here? https://llvm.org/docs/LangRef.html#llvm-vscale-intrinsic

Reply:
Good idea, added it to the docstring :)